.expeditor/config.yml

```diff
@@ -39,6 +39,8 @@ merge_actions:
       ignore_labels:
         - "Expeditor: Skip Changelog"
         - "Expeditor: Skip All"
+  - bash:.expeditor/push-git-tag.sh:
```

Comment:
The order in the config is the order the commands will be executed in, correct? I tried to confirm that in https://expeditor.chef.io/docs/getting-started/action-sets/ but I didn't see it.

If my understanding is correct, looks like BTM got this updated to run before the release.

However, the `bash` action is pre-commit by default (https://expeditor.chef.io/docs/reference/actions/#bash) so I think we need `post_commit: true` added back

Reply:
I think I actually had it here originally, but with the `post-commit: true` and expeditor had errored saying that the actions had to be listed in the order they would be executed, which had caused me to move it to the end.

It seems unlikely that the `trigger_pipeline:omnibus/release` action would be firing before a commit was finished, but it doesn't have a post-commit property on it.

I'm starting to get confused about what commit we're actually referring to.

Reply:
It should be before `trigger_pipeline`, and specified with a `post_commit: true`. I'll see if I can find out why it was giving you an error.

Reply:
I put your exact scenario into my spec tests and everything came up golden. Let's try your config with the following `merge_actions`:

```
            - bash:.expeditor/determine_version.sh:
                ignore_labels:
                  - "Expeditor: Skip Version Bump"
                  - "Expeditor: Skip All"
            - built_in:update_changelog:
                ignore_labels:
                  - "Expeditor: Skip Changelog"
                  - "Expeditor: Skip All"
            - bash:.expeditor/push-git-tag.sh:
                only_if: bash:.expeditor/determine_version.sh
                post_commit: true
            - trigger_pipeline:omnibus/release:
                ignore_labels:
                  - "Expeditor: Skip Omnibus"
                  - "Expeditor: Skip All"
                only_if: bash:.expeditor/determine_version.sh
```

Reply:
As for "what commit we're actually referring to", there is an implicit commit between the pre-commit and post-commit phases. That's likely what we're referring to.

Reply:
I probably had it between `bash:.expeditor/determine_version.sh:` and ` built_in:update_changelog:` before.